diagnostic.md

```diff
@@ -7,36 +7,36 @@ What command do we use to do that? Use the **correct** URL from your fork of
 this repository in your answer.
 
 ```sh
-<!-- Remove this comment and place your answer here. -->
+git@github.com:beckyd4234/git-basics-diagnostic.git
 ```
 
 2.What do you do after cloning a repository, before you start making any
 changes/additions?
-
-<!-- Remove this comment and place your answer here. -->
+git status
 
 3.What command do we use to create a new branch? Name this branch `response`
     in your answer. Then, how do we switch to that branch?
 
 ```sh
-<!-- Remove this comment and place your answer here. -->
+git branch response
+git checkout response
 ```
 
 4.We just wrote some code. What command do we use to see a summary of the
     changes in our working directory?
 
 ```sh
-<!-- Remove this comment and place your answer here. -->
```

Comment:
Incorrect. This should be git status. status will show a summary of all modified files (things added to be committed or changed but not yet added) 

git log shows the history of your commits.